Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -90,6 +92,12 @@ public void AddStandardLayer(LayerData layer)
             }
 
             LayerAdded.Invoke(layer);
+            layer.ParentOrSiblingIndexChanged.AddListener(OnLayerMoved);
+        }
+
+        private void OnLayerMoved(int siblingIndex, LayerData data)
+        {
+            LayerMoved.Invoke(data);
```

Comment:
Dit event wil je denk ik op RootLayer hebben als het nodig is, niet in ProjectData